mkt/constants/iarc_mappings.py

```diff
@@ -327,16 +336,130 @@
         'USK_ShopStreamingService': '',
         'USK_Tabakkonsum': 'has_usk_tobacco',
     }
-
 }
 
-# Change {body: {'key': 'val'}} to {'val': 'key'}, combining v1 and v2 dicts.
-_REVERSE_DESCS_BY_BODY = [
-    {unicode(v): unicode(k) for k, v in body_mapping.items() if v}
-    for body, body_mapping in DESCS_V2.items() + DESCS.items()]
-REVERSE_DESCS = {}
-for mapping in _REVERSE_DESCS_BY_BODY:
-    REVERSE_DESCS.update(mapping)
+# Change {body: {'key': 'val'}} to {'val': 'key'}. Used to deserialize our
+# data back to IARC.
+REVERSE_DESCS_V2 = {}
+for mapping in [{unicode(v): unicode(k) for k, v in body_mapping.items() if v}
+                for body, body_mapping in DESCS_V2.items()]:
+    REVERSE_DESCS_V2.update(mapping)
+
+
+HUMAN_READABLE_DESCS_AND_INTERACTIVES = {
+    # Descriptors:
+    'has_classind_criminal_acts': _(u'Criminal Acts'),
```

Comment:
this is a mistake since ESRB and rating bodies are very specific on how their text is translated. in the previous implementation, we did not try any l10n on our side which is why i recommended that we map each rating body to a language and serialize the string as is.